templates/Makefile

```diff
@@ -22,6 +22,11 @@ apply-env:
 	terraform init && \
 	terraform apply $(AUTO_APPROVE)
 
+pre-k8s:
+        @echo "Creating VPN private key..."
+        aws secretsmanager create-secret --name $(PROJECT)-$(ENVIRONMENT)-vpn-wg-privatekey-<% index .Params `randomSeed` %> --description "Create VPN private key from infrasture pre-k8s of makefile" --secret-string '{"key":"$(shell kubectl run -i --tty zero-k8s-utilities --image=commitdev/zero-k8s-utilities:0.0.3 --restart=Never -- wg genkey)"}' && kubectl delete pod/zero-k8s-utilities
```

Comment:
Maybe split this into 2 commands so you can handle any errors from `kubectl` a bit more gracefully? 
Also for description how about "Auto-generated Wireguard VPN private key"?
Since it's just a single value it may be easier to deal with it as a plain string rather than json. Then you don't need to create a json object by hand here or decode it later.

Reply:
done